### PR TITLE
Feature 153/projects frontend projectlist

### PR DESCRIPTION
**Pull request by lilian-delgado-jalafund**
[AB#153](https://dev.azure.com/funjaladev/7867a1c5-5ac7-4c7f-baff-87d740bf4fc1/_workitems/edit/153)

# Overview

- Update json variables
- Add view button
- Integrate API to projects component
- Add unit Tests

### CODE REVIEW

**Review by ela-ocana-jalafund — DISMISSED** [review]

**Comment by ela-ocana-jalafund** [comment]
For me it's good at the moment.

File: src/components/projects/Projects.jsx [hunk 1]
```diff
@@ -1,65 +1,86 @@
...
+                  fontSize: "1.2rem",
+                  fontWeight: "bold",
+                }}
+              >
+                Looks like, there are no projects
+              </Typography>
+            </div>
+          ) : (
+            projects.map((project) => (
+              <TableRow key={project.id}>
```
Comment (paola-quintanilla-jalafund) [inline src/components/projects/Projects.jsx]: In this case, I think is not necessary to use key

Comment (lilian-delgado-jalafund) [reply src/components/projects/Projects.jsx]: Ok, will change it right away

**Review by ela-ocana-jalafund — DISMISSED** [review]

**Review by paola-quintanilla-jalafund — DISMISSED** [review]

File: src/components/projects/Card.jsx [hunk 2]
```diff
@@ -2,24 +2,24 @@ import { CalendarToday, Person } from "@material-ui/icons";
...
+      <div className="detail-field">
+        <Person color="primary" style={{ fontSize: "1.3rem" }} />
+        <label className="details">{contact.name}</label>
+      </div>
+      <div className="detail-field">
+        <CalendarToday
+          color="primary"
+          style={{ fontSize: "1.2rem", marginLeft: "1px" }}
+        />
+        <label className="details">{creationDate}</label>
```
Comment (DaleneyRuben) [inline src/components/projects/Card.jsx]: please format the date maybe YYYY-MM-DD
Screenshot from 2021-07-29 17-18-04

**Review by DaleneyRuben — CHANGES_REQUESTED** [review]
The separation line should continue until the end of the card. I can tell it's a width issue please fix that.
Screenshot from 2021-07-29 17-49-06

File: src/components/projects/Card.jsx (hunk 2 above)
Comment (lilian-delgado-jalafund) [reply src/components/projects/Card.jsx]: Solved!

**Review by ela-ocana-jalafund — APPROVED** [review]

**Review by paola-quintanilla-jalafund — APPROVED** [review]

**Review by jorge-lema-jalafund — APPROVED** [review]
LGTM

**Comment by martin-callejas-jalafund** [comment]
@DaleneyRuben we're gonna address bug fixing in another branch

**Pull request merged**